### PR TITLE
Minor typo

### PR DESCRIPTION
**Pull request by dieterwalckiers**
I think `setState` should be `useState` here

### CODE REVIEW

**Comment by j-f1** [comment]
I’m pretty sure the original is right.

**Comment by dieterwalckiers** [comment]
Oh, I thought setState only existed in classes contexts. And the functional updater that's described here is part of the useState hook. This made me think this was a typo.

**Comment by j-f1** [comment]
When you call `useState` in a function component, you get back two things: the `state` and a `setState` function that is used to update that state. In most cases, you’d want to name those something that’s more relevant to your use case (i.e. `count` and `setCount`), but in this case we’re referring to the setter function in general so it makes sense to call it `setState`. However, this wording could be a little more clear than the current wording:

File: src/pages/a-complete-guide-to-useeffect/index.md [hunk 1]
```diff
@@ -980,7 +980,7 @@ We want to get rid of the `count` dependency in our effect.
   }, [count]);
 ```
 
-To do this, we need to ask ourselves: **what are we using `count` for?** It seems like we only use it for the `setCount` call. In that case, we don’t actually need `count` in the scope at all. When we want to update state based on the previous state, we can use the [functional updater form](https://reactjs.org/docs/hooks-reference.html#functional-updates) of `setState`:
+To do this, we need to ask ourselves: **what are we using `count` for?** It seems like we only use it for the `setCount` call. In that case, we don’t actually need `count` in the scope at all. When we want to update state based on the previous state, we can use the [functional updater form](https://reactjs.org/docs/hooks-reference.html#functional-updates) of `useState`:
```
Comment (j-f1) [inline src/pages/a-complete-guide-to-useeffect/index.md]:
```suggestion
To do this, we need to ask ourselves: **what are we using `count` for?** It seems like we only use it for the `setCount` call. In that case, we don’t actually need `count` in the scope at all. When we want to update state based on the previous state, we can use the [functional updater form](https://reactjs.org/docs/hooks-reference.html#functional-updates) of the setter function:
```

**Comment by dieterwalckiers** [comment]
Ok, it makes sense now. Thank you for clarifying this. And yes it's even more clear with the change you propose.

**Pull request closed without merging**